examples/tutorials/translations/hindi/Part 13b - Secure Classification with Syft Keras and TFE - Secure Model Serving.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<p>Change टेनर to टेन्सर</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2909/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>done! :)</p>